lib/tasks/email_notes_to_admins.rake

```diff
@@ -0,0 +1,20 @@
+desc 'Send email to admins on reservations with notes'
+task email_notes_to_admins: :environment do
+  # gets all reservations with notes and sends an email to the admin of the
+  # application, to alert them.
+  notes_reservations_out = Reservation.has_notes.checked_out.notes_unsent
+  notes_reservations_in = Reservation.has_notes.checked_in.notes_unsent
+  Rails.logger.info "Found #{notes_reservations_out.size} reservations "\
+                    "checked out with notes and #{notes_reservations_in.size} "\
+                    'reservations checked in with notes. Sending a reminder '\
+                    'email...'
+  unless notes_reservations_out.empty? && notes_reservations_in.empty?
```

Comment:
Just note that this is inside a task itself; if we were going to use a guard clause we'd have to move the code to a separate method.